Resources/public/js/components/automation-tab/main.js

```diff
@@ -263,11 +268,23 @@ define([
                 );
 
                 if (!data.id) {
-                    this.updateNotification(++this.notificationBadge);
+                    this.updateNotification(this.incrementNotificationBadge());
                 }
             }.bind(this));
         },
 
+        incrementNotificationBadge: function() {
+            return ++this.notificationBadge;
+        },
+
+        decrementNotificationBadge: function() {
+            if (this.notificationBadge === 0) {
```

Comment:
no that specific bug would be fixed with this https://github.com/sulu/SuluAutomationBundle/pull/6/files#diff-aaefa6ffa05231d7a9882ee4596c2ecbR96